### PR TITLE
Submitted Assignments for Week-5

### PR DESCRIPTION
**Pull request by Pranshu-Sharma**

### CODE REVIEW

File: Week-5/Assignment-5.1/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# Assignment-5.1
+
+Using Async/Await and Generators, create separate functions and achieve the same functionality. Execute 3 callback functions asynchronously, for example doTask1(), doTask2() and doTask3().
+
+## Solution
+
+```sh
+// 3 asynchronous callback functions
+
+function doTask1(val) {
```
Comment (jayesh96) [inline Week-5/Assignment-5.1/README.md]: is this solution or something else?

File: Week-5/Assignment-5.4/calculator.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    return a + b;
+  },
+  diff: function (a, b) {
+    return a - b;
+  },
+  product: function (a, b) {
+    return a * b;
+  },
+};
+module.exports = mathOperations;
```
Comment (jayesh96) [inline Week-5/Assignment-5.4/calculator.js:12]: The Naming Convention of constant is not right.

**Review by jayesh96 — CHANGES_REQUESTED** [review]

File: Week-5/Assignment-5.1/README.md (hunk 1 above)
Comment (Pranshu-Sharma) [reply Week-5/Assignment-5.1/README.md]: Actually we had to create 3 tasks that would run using using async/await and generators

File: Week-5/Assignment-5.4/calculator.js (hunk 2 above)
Comment (Pranshu-Sharma) [reply Week-5/Assignment-5.4/calculator.js:12]: This code was provided by Pesto team, I created the test file

**Pull request merged**